prime-dotnet-webapi/Controllers/EnrolleePaperSubmissionsController.cs

```diff
@@ -263,11 +263,6 @@ public async Task<ActionResult> SetEnrolleePaperSubmissionProfileCompleted(int e
         [ProducesResponseType(typeof(ApiResultResponse<EnrolleeAdjudicationDocument>), StatusCodes.Status200OK)]
         public async Task<ActionResult> GetAdjudicationDocuments(int enrolleeId)
         {
-            if (!await _enrolleePaperSubmissionService.PaperSubmissionIsEditableAsync(enrolleeId))
-            {
-                return NotFound($"No Editable Paper Submission found with Enrollee Id {enrolleeId}");
-            }
```

Comment:
If not checking enrollee existence any more, should also remove 
 attribute `[ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]`

Reply:
Should probably check for existence of the `Enrollee` only, not if it's editable